packages/frontend/src/app/submit/[invitationId]/components/SubmissionForm.tsx

```diff
@@ -0,0 +1,187 @@
+    .min(1, 'Required')
+    .url('Not a valid URL')
+    .endsWith('.svg', 'Must be an SVG image'),
+})
+
+type ServerError = { server?: never }
+
+export function SubmissionForm({ invitation }: { invitation: Invitation }) {
+  const {
+    register,
+    control,
+    handleSubmit,
+    formState: { errors, isSubmitting, isSubmitSuccessful },
+    setError,
+  } = useForm<Inputs & ServerError>({
+    resolver: zodResolver(schema),
+    defaultValues: {
+      role: ['verifier'],
+    },
+  })
+  async function onSubmit(data: Inputs) {
+    try {
+      await submit(data, invitation.id)
+    } catch (error: any) {
+      if (error instanceof FieldError) {
+        setError(error.field as keyof Inputs, {
+          type: 'manual',
+          message: error.message,
+        })
+      } else {
+        setError('server', { type: 'manual', message: 'Something went wrong' })
+      }
+    }
+  }
+
+  return !isSubmitSuccessful ? (
+    <div>
+      <p className="text-2xl font-bold pb-2">Submission Form</p>
+      <p className="text-sm mb-8">
+        You are submitting this form as {invitation.emailAddress}{' '}
+      </p>
+      <div className="flex flex-col gap-4">
+        <TextInput
+          type="text"
+          name="name"
+          label="Entity Name"
+          placeholder="Name"
+          register={register}
+          error={errors.name}
+        />
+        <TextArea
+          label="DIDs (split by newline)"
+          placeholder="DIDs"
+          control={control}
+          name="dids"
+          errors={errors.dids}
+        />
+        <TextInput
+          type="text"
+          name="domain"
+          label="Domain"
+          placeholder="Domain"
+          register={register}
+          error={errors.domain}
+        />
+        <Checkbox
+          options={[
+            {
+              label: 'Verifier',
+              value: 'verifier',
+              always: true,
+              disabled: true,
+            },
+
+            { label: 'Issuer', value: 'issuer' },
+          ]}
+          name="role"
+          label="Role"
+          placeholder="Role"
+          register={register}
+          errors={errors.role}
+        />
+        <TextArea
+          label="Schema IDs (split by newline)"
+          placeholder="Schema IDs"
+          control={control}
+          name="credentials"
+          errors={errors.credentials}
+        />
+        <TextInput
+          type="text"
+          name="logo_url"
+          label="Logo URL (SVG Format)"
+          placeholder="Logo URL"
+          register={register}
+          error={errors.logo_url}
+        />
+      </div>
+      <div className="card-actions justify-center mt-8">
+        <Button
+          title="Submit"
+          onClick={handleSubmit(onSubmit)}
+          loading={isSubmitting}
+        />
+      </div>
+    </div>
+  ) : (
+    <div className="flex flex-col justify-center items-center text-primary">
+      <Success />
+      <div className="mt-8 ">
```

Comment:
I wonder why Prettier doesn't complain about these spaces 🤷‍♂️ 
```suggestion
      <div className="mt-8">
```